.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
   - "4"
+  - "6"
+  - "8"
+  - "10"
```

Comment:
dynalite doesn't work with node 10

Reply:
i have no idea how this happened .. but who cares about travis.yml?